src/index.jsx

```diff
@@ -14,11 +14,23 @@ import initializeStore from './app/store';
 import CourseView from './features/course-view/CourseView';
 import Layout from './features/layout/Layout';
 
+ensureConfig([
+  'DISABLE_APP_HEADER',
+], 'CoursePlayer component');
+
+ensureConfig([
```

Comment:
This can be merged with the previous ensureConfig call, just add your name to the array

Reply:
The seconf string is for identifying where the issue is if the variable was missing, you will get a log with something like "ENABLE_PENDO config missing in CoursePlayer component"